Sources/XCTest/XCTestCase.swift

```diff
@@ -8,12 +8,58 @@
 //
 //
 //  XCTestCase.swift
-//  Base protocol (and extension with default methods) for test cases
+//  Base class for test cases
 //
 
-public protocol XCTestCase : XCTestCaseProvider {
-    func setUp()
-    func tearDown()
+public typealias XCTestMethod = XCTestCase throws -> Void
```

Comment:
I've added a couple more doc entries to public symbols now, just for you :smile: